index.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+$db = new PDO('mysql:host=db; dbname=collections_display', 'root', 'password');
+
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$query = $db->prepare('SELECT * FROM `board_games`;');
+
+$query->execute();
+
+$result = $query->fetchAll();
+
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <style>
```

Comment:
Same here, move all of your CSS into a single css file and link to it from both of your php files

Reply:
Done